components/transaction_manager.py

```diff
@@ -1,10 +1,8 @@
-from decouple import config
-
 from services import LoadContracts
 from web3 import gas_strategies
 from web3.gas_strategies import time_based
 
-from utilities import constants
+from utilities import cruize_constants
```

Comment:
why did we rename this to cruize_constants ? constants was fine no ?

Reply:
it will create confusion  because  dydx use constant .py for their constant